eslint.config.mjs

```diff
@@ -41,7 +41,7 @@ export default [
     languageOptions: {
       sourceType: "script",
       parserOptions: {
-        project: true,
+        projectService: true,
```

Comment:
https://typescript-eslint.io/blog/announcing-typescript-eslint-v8-beta#project-service explains the newly stable "project service" option.

Interestingly, when I left this as `project: true` and ran linting, a bunch of `.vue` files were reported as not having proper type info:

```plaintext

/Users/josh/repos/eslint-plugin-yml/docs/.vitepress/theme/components/components/EslintPluginEditor.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/EslintPluginEditor.vue

/Users/josh/repos/eslint-plugin-yml/docs/.vitepress/theme/components/components/PgEditor.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/PgEditor.vue

/Users/josh/repos/eslint-plugin-yml/docs/.vitepress/theme/components/components/RulesSettings.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/RulesSettings.vue

/Users/josh/repos/eslint-plugin-yml/docs/.vitepress/theme/components/components/SnsBar.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/SnsBar.vue

/Users/josh/repos/eslint-plugin-yml/docs/.vitepress/theme/components/eslint-code-block.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/eslint-code-block.vue

/Users/josh/repos/eslint-plugin-yml/docs/.vitepress/theme/components/playground-block.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/playground-block.vue
```

...I don't know why and haven't dug into investigate.

Reply:
This should be fixed by https://github.com/typescript-eslint/typescript-eslint/pull/9580.